dev/lint-python

```diff
@@ -96,6 +98,32 @@ fi
 
 rm "$PEP8_REPORT_PATH"
 
+# Check that the documentation builds acceptably, skip check if sphinx is not installed.
+if hash "$SPHINXBUILD" 2> /dev/null; then
+  cd python/docs
+  make clean
+  export SPHINXOPTS=""
+  make linkcheck &>> "$SPHINX_REPORT_PATH"  || lint_status=1
```

Comment:
I did that for append and I don't get that error in bash 4.3.11 but we can add remove it since we aren't going to do linkcheck we can just overwrite.